<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>358</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuQMenu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_menu_New"/>
    <addaction name="action_menu_Open"/>
    <addaction name="action_menu_Save"/>
    <addaction name="action_menu_Save_How"/>
    <addaction name="separator"/>
    <addaction name="action_menu_Exit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="action_edit_Cancel"/>
    <addaction name="action_edit_ReDo"/>
    <addaction name="action_edit_Copy"/>
    <addaction name="action_edit_Cut"/>
    <addaction name="action_edit_Paste"/>
    <addaction name="action_edit_Find"/>
    <addaction name="action_edit_FindAndChange"/>
    <addaction name="action_edit_SelectAll"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Формат</string>
    </property>
    <addaction name="action_format_WordWrap"/>
    <addaction name="action_format_SelectFont"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Вид</string>
    </property>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>Редактирование стиля подсветки</string>
     </property>
     <addaction name="action_view_Change"/>
     <addaction name="action_view_Download"/>
     <addaction name="actionDefault"/>
    </widget>
    <widget class="QMenu" name="menu_6">
     <property name="title">
      <string>Выбор Синтаксиса</string>
     </property>
     <addaction name="action_89"/>
     <addaction name="action_98"/>
     <addaction name="action_11"/>
     <addaction name="action_17"/>
    </widget>
    <addaction name="action_view_Background"/>
    <addaction name="action_view_lineColor"/>
    <addaction name="separator"/>
    <addaction name="action_view_NumLine"/>
    <addaction name="action_view_ToolBar"/>
    <addaction name="action_view_LineState"/>
    <addaction name="action_view_SintBacklight"/>
    <addaction name="separator"/>
    <addaction name="menu_6"/>
    <addaction name="menu_5"/>
    <addaction name="action_ini"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menuQMenu"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionCancel"/>
   <addaction name="actionReDo"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionFind"/>
   <addaction name="actionFindAndChange"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/img/img/new.ico</normaloff>:/img/img/new.ico</iconset>
   </property>
   <property name="text">
    <string>Новый Документ</string>
   </property>
   <property name="toolTip">
    <string>Создать новый документ, Ctrl+N</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/img/img/open.ico</normaloff>:/img/img/open.ico</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="toolTip">
    <string>Открыть готовый документ, Ctrl+O</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/img/img/save.ico</normaloff>:/img/img/save.ico</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="toolTip">
    <string>Сохранить текущий документ, Ctrl+S</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCancel">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/img/img/cancel.ico</normaloff>:/img/img/cancel.ico</iconset>
   </property>
   <property name="text">
    <string>Отменить</string>
   </property>
   <property name="toolTip">
    <string>Отменить последнее действие</string>
   </property>
  </action>
  <action name="actionReDo">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/img/img/back.ico</normaloff>:/img/img/back.ico</iconset>
   </property>
   <property name="text">
    <string>Повторить</string>
   </property>
   <property name="toolTip">
    <string>Повторить предыдущее действие</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/img/img/copy.ico</normaloff>:/img/img/copy.ico</iconset>
   </property>
   <property name="text">
    <string>Копировать</string>
   </property>
   <property name="toolTip">
    <string>Копировать</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/img/img/cut.ico</normaloff>:/img/img/cut.ico</iconset>
   </property>
   <property name="text">
    <string>Вырезать</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/img/img/insert.ico</normaloff>:/img/img/insert.ico</iconset>
   </property>
   <property name="text">
    <string>Вставить</string>
   </property>
   <property name="toolTip">
    <string>Вставить</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/img/img/find.ico</normaloff>:/img/img/find.ico</iconset>
   </property>
   <property name="text">
    <string>Найти</string>
   </property>
   <property name="toolTip">
    <string>Найти</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionFindAndChange">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/img/img/find next.ico</normaloff>:/img/img/find next.ico</iconset>
   </property>
   <property name="text">
    <string>Найти и заменить</string>
   </property>
   <property name="toolTip">
    <string>Найти и заменить</string>
   </property>
  </action>
  <action name="action_menu_New">
   <property name="text">
    <string>Новый</string>
   </property>
  </action>
  <action name="action_menu_Open">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="action_menu_Save">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_menu_Save_How">
   <property name="text">
    <string>Сохранить как</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_menu_Exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_edit_Cancel">
   <property name="text">
    <string>Отменить</string>
   </property>
  </action>
  <action name="action_edit_ReDo">
   <property name="text">
    <string>Повторить</string>
   </property>
  </action>
  <action name="action_edit_Copy">
   <property name="text">
    <string>Копировать</string>
   </property>
  </action>
  <action name="action_edit_Cut">
   <property name="text">
    <string>Вырезать</string>
   </property>
  </action>
  <action name="action_edit_Paste">
   <property name="text">
    <string>Вставить</string>
   </property>
  </action>
  <action name="action_edit_Find">
   <property name="text">
    <string>Найти</string>
   </property>
  </action>
  <action name="action_edit_FindAndChange">
   <property name="text">
    <string>Найти и заменить</string>
   </property>
  </action>
  <action name="action_edit_SelectAll">
   <property name="text">
    <string>Выделить все</string>
   </property>
  </action>
  <action name="action_format_WordWrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Перенос по словам</string>
   </property>
  </action>
  <action name="action_format_SelectFont">
   <property name="text">
    <string>Выбор шрифа</string>
   </property>
  </action>
  <action name="action_view_Background">
   <property name="text">
    <string>Выбор цвета фона</string>
   </property>
  </action>
  <action name="action_view_NumLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Вкл/Выкл отображения нумерации строк</string>
   </property>
  </action>
  <action name="action_view_ToolBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Вкл/Выкл отображения панели инструментов</string>
   </property>
  </action>
  <action name="action_view_LineState">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Вкл/Выкл отображения строки состояния</string>
   </property>
  </action>
  <action name="action_view_SintBacklight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Вкл/Выкл подсветки синтаксиса</string>
   </property>
  </action>
  <action name="action_view_Change">
   <property name="text">
    <string>Изменить</string>
   </property>
  </action>
  <action name="action_view_Download">
   <property name="text">
    <string>Загрузка стиля из файла</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actionStatus">
   <property name="text">
    <string>Status</string>
   </property>
  </action>
  <action name="action_view_lineColor">
   <property name="text">
    <string>Выбор цвета текущей строки</string>
   </property>
  </action>
  <action name="action_89">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Си89</string>
   </property>
  </action>
  <action name="action_98">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Си++98</string>
   </property>
  </action>
  <action name="action_11">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Си++11</string>
   </property>
  </action>
  <action name="action_17">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Си++20</string>
   </property>
  </action>
  <action name="action_ini">
   <property name="text">
    <string>Сохранить настройки в ini-файл</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
